### PR TITLE
ENH Customisable title link

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-session-manager/issues/38

Userhelp link is 'TODO' - though there's a follow up issue to add this in when userhelp has been created https://github.com/silverstripe/silverstripe-session-manager/issues/15

### CODE REVIEW

File: src/Extensions/MemberExtension.php [hunk 1]
```diff
@@ -40,8 +40,10 @@ public function updateCMSFields(FieldList $fields)
             'Root.Main',
             $sessionManagerField = SessionManagerField::create(
                 'SessionManagerField',
-                _t(__CLASS__ . '.SESSION_MANAGER_SETTINGS_FIELD_LABEL', 'Devices'),
-                $this->owner->ID
+                _t(__CLASS__ . '.DEVICES', 'Devices'),
+                $this->owner->ID,
+                _t(__CLASS__ . '.LEARN_MORE', 'Learn more'),
+                'http://userhelp.silverstripe.org/TODO'
```
Comment (bergice) [inline src/Extensions/MemberExtension.php]: Link needs updating.

File: templates/SilverStripe/SessionManager/FormField/SessionManagerField_holder.ss [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<div id="$HolderID" class="form-group field<% if $extraClass %> $extraClass<% end_if %>">
+    <% if $Title %>
+        <label for="$ID" id="title-$ID" class="form__field-label">
+            $Title
+            <% if $TitleLinkHref && $TitleLinkText %>
+                <br><a href="$TitleLinkHref" target="_blank">$TitleLinkText</a>
+            <% end_if %>
+        </label>
+    <% end_if %>
+    <div>$Title</label>
```
Comment (bergice) [inline templates/SilverStripe/SessionManager/FormField/SessionManagerField_holder.ss]: Should this be removed?

```suggestion
```

Comment (emteknetnz) [reply templates/SilverStripe/SessionManager/FormField/SessionManagerField_holder.ss]: lol yup, copy paste error

File: src/Extensions/MemberExtension.php (hunk 1 above)
Comment (emteknetnz) [reply src/Extensions/MemberExtension.php]: No userhelp to link to right now =)  I've added it as an AC on the [userhelp issue](https://github.com/silverstripe/silverstripe-session-manager/issues/15)

**Review by bergice — APPROVED** [review]
LGTM, will need a rebase.

**Comment by emteknetnz** [comment]
@bergice Has been rebased and travis green, should be good for merge

**Pull request merged**